core/src/block_sync.rs

```diff
@@ -129,193 +101,140 @@ impl BlockSynchronizer {
-            Self { hash, peer_id }
-        }
+impl GetBlocksAfter {
+    /// Construct [`GetBlocksAfter`].
+    pub const fn new(hash: HashOf<VersionedCommittedBlock>, peer_id: PeerId) -> Self {
+        Self { hash, peer_id }
     }
+}
 
-    /// Message variant to share blocks to peer
-    #[derive(Debug, Clone, Decode, Encode)]
-    pub struct ShareBlocks {
-        /// Blocks
-        pub blocks: Vec<VersionedCommittedBlock>,
-        /// Peer id
-        pub peer_id: PeerId,
-    }
+/// Message variant to share blocks to peer
+#[derive(Debug, Clone, Decode, Encode)]
+pub struct ShareBlocks {
+    /// Blocks
+    pub blocks: Vec<VersionedCommittedBlock>,
+    /// Peer id
+    pub peer_id: PeerId,
+}
 
-    impl ShareBlocks {
-        /// Construct [`ShareBlocks`].
-        pub const fn new(blocks: Vec<VersionedCommittedBlock>, peer_id: PeerId) -> Self {
-            Self { blocks, peer_id }
-        }
+impl ShareBlocks {
+    /// Construct [`ShareBlocks`].
+    pub const fn new(blocks: Vec<VersionedCommittedBlock>, peer_id: PeerId) -> Self {
+        Self { blocks, peer_id }
     }
+}
 
-    /// Message's variants that are used by peers to communicate in the process of consensus.
-    #[version_with_scale(n = 1, versioned = "VersionedMessage")]
-    #[derive(Debug, Clone, Decode, Encode, FromVariant, iroha_actor::Message)]
-    pub enum Message {
-        /// Request for blocks after the block with `Hash` for the peer with `PeerId`.
-        GetBlocksAfter(GetBlocksAfter),
-        /// The response to `GetBlocksAfter`. Contains the requested blocks and the id of the peer who shared them.
-        ShareBlocks(ShareBlocks),
-    }
+/// Message's variants that are used by peers to communicate in the process of consensus.
+#[version_with_scale(n = 1, versioned = "VersionedMessage")]
+#[derive(Debug, Clone, Decode, Encode, FromVariant, iroha_actor::Message)]
+pub enum Message {
+    /// Request for blocks after the block with `Hash` for the peer with `PeerId`.
+    GetBlocksAfter(GetBlocksAfter),
+    /// The response to `GetBlocksAfter`. Contains the requested blocks and the id of the peer who shared them.
+    ShareBlocks(ShareBlocks),
+}
 
-    impl Message {
-        /// Handles the incoming message.
-        #[iroha_futures::telemetry_future]
-        pub async fn handle(&self, block_sync: &mut BlockSynchronizer) {
-            match self {
-                Message::GetBlocksAfter(GetBlocksAfter { hash, peer_id }) => {
-                    if block_sync.block_batch_size == 0 {
-                        warn!("Error: not sending any blocks as batch_size is equal to zero.");
-                        return;
-                    }
-                    if *hash == block_sync.sumeragi.latest_block_hash() {
-                        return;
-                    }
+impl Message {
+    /// Handles the incoming message.
+    #[iroha_futures::telemetry_future]
+    pub async fn do_message(&self, block_sync: &mut BlockSynchronizer) {
```

Comment:
`do` isn't very informative. Please consider renaming.

Reply:
It's not that different from "handle" but sure.